assets/js/localization.js

```diff
@@ -9,9 +9,16 @@ var dynamicLocalizations = {
     'detected': 'detected',
     'File_Too_Large': 'File is too large!',
     'Format_Not_Supported': 'Format not supported!',
-    'Download_File': 'Download {{fileName}}'
+    'Download_File': 'Download {{fileName}}',
+    'Suggest_Sentence': 'How would you suggest we translate {{targetWordCode}}?',
+    'Suggest_Title': 'Improve Apertium\'s translation',
+    'Suggest_Placeholder': 'New word'
 };
 
+var localizeRecaptcha = ['ar', 'af', 'am', 'hy', 'az', 'eu', 'bn', 'bg', 'ca', 'zh-HK', 'zh-CN', 'zh-TW', 'hr', 'cs', 'da', 'nl', 'en-GB', 'en', 'et', 'fil', 'fi', 'fr', 'fr-CA', 'gl', 'ka', 'de', 'de-AT', 'de-CH', 'el', 'gu', 'iw', 'hi', 'hu', 'is', 'id', 'it', 'ja', 'kn', 'ko', 'lo', 'lv', 'lt', 'ms', 'ml', 'mr', 'mn', 'no', 'fa', 'pl', 'pt', 'pt-BR', 'pt-PT', 'ro', 'ru', 'sr', 'si', 'sk', 'sl', 'es', 'es-419', 'sw', 'sv', 'ta', 'te', 'th', 'tr', 'uk', 'ur', 'vi', 'zu'];
```

Comment:
Please add `Languages` to these variable names, `localizeRecaptchaLanguages` and `localizeReacaptchaAlternativeLanguages` and move them to after L4 (no line between them either) :)